src/Common/src/CoreLib/System/Resources/ResourceReader.core.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Resources
+{
+    partial class ResourceReader
```

Comment:
>partial [](start = 4, length = 7)

nit: add private/internal specifier?

Reply:
I intentionally left it off the partial class because I wanted it to only be specified once.  Do we have guidelines that say to make it match?  I know its unlikely to change, but I prefer to follow the [dry](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle when I can.

Reply:
It looks you are right. Looking at the example https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md and I am seeing it is doing what you are doing. I am not sure this was intentional though.

Reply:
It was intentional in the guidelines because it's avoiding duplication of the interface declarations.  I'm fine either way and will defer to your judgement since you own these types.

Reply:
I am fine to keep what you have.